packages/gatsby-plugin-subfont/src/gatsby-node.js

```diff
@@ -1,22 +1,23 @@
 const path = require(`path`)
-const { execSync } = require(`child_process`)
-const shellescape = require(`shell-escape`)
+const subfont = require(`subfont`)
 
-exports.onPostBuild = ({ store }) => {
+exports.onPostBuild = async ({ store }, options) => {
   const root = path.join(store.getState().program.directory, `public`)
-  // TODO make this configurable
+
   const urlPaths = [`/`]
-  const baseArgs = [
-    `node_modules/.bin/subfont`,
-    `-i`,
-    `--no-recursive`,
-    `--inline-css`,
-    `--root`,
-    `file://${root}`,
-  ]
-  const args = baseArgs.concat(
-    urlPaths.map(currentPath => path.join(root, currentPath, `index.html`))
+
+  await subfont(
+    {
+      root: `file://${root}`,
```

Comment:
The `file://` protocol looks a little funny to me, the docs for `root` just mention it being a path.

Reply:
Fixed, thanks.